docs/en/remote-flow.rst

```diff
@@ -47,26 +48,30 @@ The details of each step shown in the previous picture are described in the tabl
   * - **Id**
     - **Description**
   * - **1**, **2**
-    - The User requests to access to a protected resource, the Relying Party redirects the User to a discovery page in which the User selects the *Login with the Wallet* button. The Authorization flow starts.
+    - The User requests to access to a protected resource of the Relying Party.
   * - **3**, **4**,
-    - The Relying Party provides the Wallet Instance with a URL where a generic signed Request Object can be downloaded.
+    - The Relying Party provides the Wallet Instance with a URL where the information about the Relying Party are provided, along with the information about where the signed request is available for download.
   * - **5**, **6**, **7**, **8**, **9**
-    - In the **Cross Device Flow**: the Request URI is provided in the form of a QR Code that is shown to the User. The User frames the QRCode with the Wallet Instance and extracts  ``client_id``, ``request_uri`` and ``state``. In the **Same Device Flow** the Relying Party provides the same information of the Cross-Device flow but in the form of HTTP Redirect Location (302). 
-  * - **10**, **11**, **12**
+    - In the **Cross Device Flow**: the Request URI is provided in the form of a QR Code that is shown to the User. The User frames the QRCode with the Wallet Instance that extracts  ``client_id``, ``request_uri``, ``state``, ``client_id_scheme`` and ``request_uri_method``. In the **Same Device Flow** the Relying Party provides the same information of the Cross-Device flow but in the form of HTTP Redirect Location (302). 
```

Comment:
In the same device flow it is always the same URL contained in the QR-Code but which is opened by the browser (instead of using the camera). Then it is the operating system that "interprets" that URL and opens the Wallet Instance. There should be no 302 redirect

Reply:
done